.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+on: ["push", "pull_request"]
```

Comment:
Do we actually need "push"?

Reply:
I think so, just in case someone *"accidentally push to master"*

Reply:
ftr, we could protect the `master` branch in the projects settings to stop them doing that.

Reply:
fixed, I will change `master`'s settings, thanks